### PR TITLE
Add unit test for useActiveSpeakersState

### PR DESCRIPTION
**Pull request by ltrung**
**Description of changes:**
- Add `@testing-library/react-hooks` dev dependency to support testing hooks.
- Add a sample unit test for `useActiveSpeakersState`.

**Testing**
1. Have you successfully run `npm run build:release` locally? Yes

2. How did you test these changes? Manually run the test.

3. If you made changes to the component library, have you provided corresponding documentation changes? No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tst/hooks/sdk/useActiveSpeakersState.test.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  };
+});
+
+beforeEach(() => {
+  // Clear all instances and calls to constructor and all methods:
+  mockMeetingManager.mockClear();
+});
+
+it('should use active speaker', () => {
+  expect(mockMeetingManager).toHaveBeenCalledTimes(0);
```
Comment (nainkunal933) [inline tst/hooks/sdk/useActiveSpeakersState.test.tsx:29]: This expect should be a separate test: `should not call meeting manager before use active speaker`

File: tst/hooks/sdk/useActiveSpeakersState.test.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  const { result, unmount } = renderHook(() => useActiveSpeakersState(), {
+    wrapper: ({ children }) => <MeetingContext.Provider value={new MeetingManager({ logLevel: LogLevel.OFF})}>{children}</MeetingContext.Provider>
+  })
+  expect(mockMeetingManager).toHaveBeenCalledTimes(1);
+  expect(subscribeToActiveSpeakerSpy).toHaveBeenCalledTimes(1);
+  expect(result.current).toStrictEqual([]);
+  unmount();
```
Comment (nainkunal933) [inline tst/hooks/sdk/useActiveSpeakersState.test.tsx:36]: Technically, this `unmount` and the associated assert should be a separate test

File: tst/hooks/sdk/useActiveSpeakersState.test.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    subscribeToActiveSpeaker: subscribeToActiveSpeakerSpy,
+    unsubscribeFromActiveSpeaker: unsubscribeToActiveSpeakerSpy
+  };
+});
+
+beforeEach(() => {
+  // Clear all instances and calls to constructor and all methods:
+  mockMeetingManager.mockClear();
+});
+
```
Comment (nainkunal933) [inline tst/hooks/sdk/useActiveSpeakersState.test.tsx:27]: Can you please add a `describe` block here for the `it` statement? It makes reading the coverage reports much easier.

**Review by nainkunal933 — COMMENTED** [review]
Thank you for adding this test. My main concern with the test is that the unit test seems to be testing more than a single unit. It can broken down.

File: tst/hooks/sdk/useActiveSpeakersState.test.tsx (hunk 3 above)
Comment (ltrung) [reply tst/hooks/sdk/useActiveSpeakersState.test.tsx:28]: Fixed.

**Review by xuesichao — APPROVED** [review]

**Pull request merged**